src/psyclone/tests/test_files/gocean1p0/test27_loop_swap.f90

```diff
@@ -0,0 +1,27 @@
+!-------------------------------------------------------------------------------
```

Comment:
Please could you use the same copyright header as in the latest dynamo test files? Also, you are the author, not me :-)